modules/nf-core/motus/preplong/meta.yml

```diff
@@ -0,0 +1,50 @@
+name: "motus_preplong"
+description: Taxonomic meta-omics profiling using universal marker genes
+keywords:
+  - classify
+  - metagenomics
+  - fastq
+  - taxonomic profiling
+tools:
+  - "motus":
+      description: "Marker gene-based OTU (mOTU) profiling"
```

Comment:
What did OTU mean?

Reply:
It mean operational taxonomic unit.

Reply:
Can you please elaborate that in the meta then

Reply:
Done!